tensorboard/plugins/debugger/tf_debugger_dashboard/tf-debugger-dashboard.html

```diff
@@ -932,6 +950,15 @@
           graph.panToNode(nodeName);
         } else {
           // Selecting the node triggers a pan to it.
+          const nodeMap = graph.get('renderHierarchy').hierarchy.getNodeMap();
+          if (nodeMap[nodeName] == null) {
```

Comment:
Maybe just `!nodeMap[nodeName]`? It might be `undefined`.

Reply:
Done. Thanks for the review.